technical-specs/index.md

```diff
@@ -11,125 +11,129 @@ permalink: /technical-specs/index.html
+                        <ul>
+                            <li>{% t specs.block_emission_main %}</li>
+                            <li>{% t specs.block_emission_tail %}</li>
+                            <li>{% t specs.block_emission_disclaimer1 %} <a href="https://www.reddit.com/r/Monero/comments/512kwh/useful_for_learning_about_monero_coin_emission/">{% t specs.block_emission_disclaimer_link %}</a>{% t specs.block_emission_disclaimer2 %}</li>
+                        </ul>
                     </div>
                     <div class="row">
                         <div class="col">
-                            <h3>Max supply</h3>
+                            <h3>{% t specs.supply_title %}</h3>
                         </div>
                     </div>
                     <div>
-                        <ol>
-                            <li>infinite</li>
-                        </ol>
+                        <ul>
+                            <li>{% t specs.supply_amount %}</li>
+                        </ul>
                     </div>
                     <div class="row">
                         <div class="col">
-                            <h3>Sender privacy</h3>
+                            <h3>{% t specs.sender_privacy_title %}</h3>
                         </div>
                     </div>
                     <div>
-                        <ol>
-                            <li>Ring signatures</li>
-                        </ol>
+                        <ul>
+                            <li>{% t specs.sender_privacy_mode %}</li>
+                        </ul>
                     </div>
                     <div class="row">
                         <div class="col">
-                            <h3>Recipient privacy</h3>
+                            <h3>{% t specs.recipient_privacy_title %}</h3>
                         </div>
                     </div>
                     <div>
-                        <ol>
-                            <li>Stealth addresses</li>
-                        </ol>
+                        <ul>
+                            <li>{% t specs.recipient_privacy_mode %}</li>
+                        </ul>
                     </div>
                     <div class="row">
                         <div class="col">
-                            <h3>Amount obfuscation</h3>
+                            <h3>{% t specs.amount_hidden_title %}</h3>
                         </div>
                     </div>
                     <div>
-                        <ol>
-                            <li>Ring confidential transactions</li>
-                        </ol>
+                        <ul>
+                            <li>{% t specs.amount_hidden_mode %}</li>
+                        </ul>
                     </div>
                 </div>
             </div>
             <!-- end right one-third block-->
         </div>
     </section>
 </div>
+
+<div class="untranslated {% t monero-project.translated %}">
```

Comment:
Change this to read `{% t specs.translated %}` otherwise this pulls from the monero project page.

Reply:
Oh yes. Sorry. I didn't notice.